client/config/signup.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
@dropcat13 this file also seems like it should be in the components directory???